src/operator/edgeHandlingMethod.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * Created by Cristian on 17/07/2015.
+ */
+export default function edgeHandlingMethod(newImage, method, k){
```

Comment:
split in two methods:
* copyEdgeHandler
* blackEdgeHandler
(and after the "mirrorEdgeHandler")

You can completely remove the method checks, it should be done in `convolution` when the options are processed to select the right method.